lua/packer.lua

```diff
@@ -521,6 +521,9 @@ packer.update = function(...)
     log.debug 'Gathering update tasks'
     await(a.main)
     update_tasks, display_win = update(plugins, installed_plugins, display_win, results, opts)
+    if update_tasks == nil then
+      return
+    end
```

Comment:
Same comment and suggestion: to account for cases with *only* Luarocks tasks, I think we cannot exit early here.